### DISCUSSION TITLE
Nested Comments

### DISCUSSION THREAD

**Discussion by orenelbaum** [DanielXMoore/Civet#291] [Ideas]
```js
/*
hello
/*
there
*/
const commentedOut = 1
*/

const hello = 1
```

output (potentially):

```js
/*
hello
/*
there
*\/
const commentedOut = 1
*/

const hello = 1
```

**Comment by edemaine** [comment]
As recently discussed in #288.  I think it could be helpful for commenting out blocks.  We'll probably want to turn it off in `jsCompat` mode though.

**Reply by orenelbaum** [reply]
Yeah I'm sure it will be useful, I come across this pretty often.
